opencti-platform/opencti-front/src/private/components/drafts/DraftContextBanner.tsx

```diff
@@ -116,6 +120,9 @@ const DraftContextBannerComponent: FunctionComponent<DraftContextBannerComponent
             onCompleted: () => {
               setApproving(false);
               MESSAGING$.notifySuccess('Draft validation in progress');
+              if (entity_id) {
+                navigate(`/dashboard/id/${entity_id}`);
+              }
               navigate('/dashboard/data/import/draft');
             },
```

Comment:
Same as above, should be rewritten as
```suggestion
              if (entity_id) {
                navigate(`/dashboard/id/${entity_id}`);
              }
              else {
                 navigate('/dashboard/data/import/draft');
              }
            },
```